src/sentry/event_manager.py

```diff
@@ -2226,7 +2238,7 @@ def _get_severity_metadata_for_group(
         }
     except Exception as e:
         logger.warning("Failed to calculate severity score for group", repr(e))
-
+        cache.set(SEER_ERROR_COUNT_KEY, cache.get(SEER_ERROR_COUNT_KEY, 0) + 1, timeout=60 * 60)
```

Comment:
@snigdhas It's possible the data we fetch is stale when we set it, is there an incrementor option?

Reply:
yeah we can use [cache.incr](https://docs.djangoproject.com/en/5.0/topics/cache/#django.core.cache.cache.incr) but afaict that doesn't have a way to set a ttl

Reply:
I think we should use `incr`, and maybe just set the ttl separately.

Another option here is to use redis directly, and use a pipeline to increment and then set the ttl in one request.

Reply:
@wedamija Isn't memcached here better because of higher availability?

Reply:
They're about the same, redis is used in pretty important places like https://github.com/getsentry/sentry/blob/7072fed97d85b61a2d1869ad5d8cf578e3a67287/src/sentry/tasks/post_process.py#L578, which stores all incoming events as part of ingest.

We should just make sure we're on a redis cluster connection instead of redis blaster

Reply:
How do we make sure of that?

Reply:
If we're going to keep using `cache` here it doesn't matter too much, but you can check which clusters are set up in the getsentry config

Reply:
I think the read/write load we're adding here should be fine to keep using `cache`, wdyt?

Reply:
Yeah seems fine to me